crates/pink/pink-extension/src/system.rs

```diff
@@ -136,6 +136,12 @@ pub trait System {
     /// Get the history of given driver.
     #[ink(message)]
     fn driver_history(&self, name: String) -> Option<Vec<(crate::BlockNumber, AccountId)>>;
+
+    /// Get current event chain head info
+    ///
+    /// Returns (next event block number, last_event_block_hash)
```

Comment:
Either natural language or snake case variable name. Let's don't mix the both.

Reply:
Fixed